tests/libnet/network_attachment_definition.go

```diff
@@ -26,13 +26,24 @@ import (
 	"kubevirt.io/kubevirt/tests/framework/kubevirt"
 )
 
-const postUrl = "/apis/k8s.cni.cncf.io/v1/namespaces/%s/network-attachment-definitions/%s"
+const PasstNetAttDef = "netbindingpasst"
 
 func CreateNetworkAttachmentDefinition(name, namespace, netConf string) error {
+	const postUrl = "/apis/k8s.cni.cncf.io/v1/namespaces/%s/network-attachment-definitions/%s"
 	return kubevirt.Client().RestClient().
 		Post().
 		RequestURI(fmt.Sprintf(postUrl, namespace, name)).
 		Body([]byte(netConf)).
 		Do(context.Background()).
 		Error()
 }
+
+func CreatePasstNetworkAttachmentDefinition(namespace string) error {
```

Comment:
I'm not clear how this change is related to the commit description. Either it got here by mistake or the commit needs to be updated.

Reply:
done - changed the commit description and removed the passt registration method.